addOns/wappalyzer/src/test/resources/org/zaproxy/zap/extension/wappalyzer/apps.json

```diff
@@ -44,6 +44,16 @@
       "description": "Test Entry is a test entry for UnitTests",
       "url": "^https?://test\\.example\\.com",
       "website": "https://www.example.com/testentry"
+    },
+     "Test Entry2": {
+      "cats": [
+        36
+      ],
+    "dom": 
+      "script[src*='sites/g/files']",
```

Comment:
Could this be reformatted? It's hard to read.

Reply:
You mean just to put it on the same line?

Reply:
yes, and indent to the right.